docs/assets/funfacts.txt

```diff
@@ -16,6 +16,21 @@ SkyClient was once exclusive to Windows users.\nFortunately, now we have a Java-
 You'll never believe this, but our Discord's support channels\nare in a category called SUPPORT IS HERE! Use them as necessary.
 SBE has skidded from countless mods including DSM, Skytils and NEU!
 SkyClient contains only the finest ethically sourced mods and packs -\nwe asked permission before adding them to SkyClient.
+In Canada, Santa's postal code is H0H 0H0.
+Zetvue makes music! You can too!
+Guvf vf abg na vzcbegnag fragrapr.
+ROT13
+Vs lbh qrpvcure guvf, lbh fubhyq yvfgra gb Mrgihr'f zhfvp.
```

Comment:
you know these aren't displayed in order or anything

Reply:
i know

Reply:
well it's going to be really confusing

Reply:
that's the point
anyone that knows ROT13 will know lol

Reply:
```suggestion
Guvf vf abg na vzcbegnag fragrapr.
Vs lbh qrpvcure guvf, lbh fubhyq yvfgra gb Mrgihr'f zhfvp.
```
also you only get one ad